contracts/protocol/clients/voucher/BosonVoucher.sol

```diff
@@ -284,18 +277,14 @@ contract BosonVoucherBase is IBosonVoucher, BeaconClientBase, OwnableUpgradeable
         (Offer memory offer, OfferDates memory offerDates) = getBosonOffer(_offerId);
         require(offer.voided || (offerDates.validUntil <= block.timestamp), OFFER_STILL_VALID);
 
-        // Get max amount that can be burned in a single transaction
-        address protocolDiamond = IClientExternalAddresses(BeaconClientLib._beacon()).getProtocolAddress();
-        uint256 maxPremintedVouchers = IBosonConfigHandler(protocolDiamond).getMaxPremintedVouchers();
-
         // Get the first token to burn
         uint256 start = (range.lastBurnedTokenId == 0) ? range.start : (range.lastBurnedTokenId + 1);
 
         // Get the last token to burn
-        uint256 end = range.start + range.minted;
+        uint256 end = start + _amount;
 
         // End should be greater than start
-        require(end > start, NOTHING_TO_BURN);
+        require(end > start && end <= range.start + range.minted, AMOUNT_EXCEEDS_RANGE_OR_NOTHING_TO_BURN);
```

Comment:
I guess the intention was to use range.start here and not `range`

Reply:
cc @anajuliabit and @zajck  ...

Reply:
I think what @anajuliabit  did is correct.
`range.start + range.minted` is the highest `tokenId` that can be burned, so it's making sure that `end` is not above that.

Reply:
yeah, thank both. I will resolve, i was just calling it out, as I wanted to make sure that it was intentional.